### PR TITLE
Move over content from nengo.ca

### PR DESCRIPTION
**Pull request by tbekolay**
This moves all the relevant content from nengo.ca here. Once this is merged in, I'll set up the nengo.ai custom domain to point here, and then we can forward nengo.ca to nengo.ai as our main Nengo homepage.

A lot of what was previously on nengo.ca is specific to Nengo 1.4 or Spaun, so those aspects will be moved to those documentation pages instead. For Nengo 1.4, a PR is [in progress](https://github.com/nengo/nengo_1.4/pull/433); a Spaun PR will be made shortly (will update this post when that happens).

Those two PRs should be merged before this one is merged, as these changes include links to pages that will only exist once those PRs are merged (particularly for Spaun).

Additionally, I've added to the dev meeting agenda that we should talk about things that we know link to pages on nengo.ca so that we can try to minimize the impact of redirecting to nengo.ai. It would be possible to set up something on nengo.ca that exists only to redirect to the appropriate new locations, but not sure if it's worth the effort if things are easy to find on the new site.

One additional difficulty in moving things over is that (to my knowledge) http://nengo.ca/publications/spaunsciencepaper is a special URL because it gives access to the full Science article without having to be logged in or from a university IP. If that is indeed the case, then we should figure out if we can do that for a different URL, or if redirecting is sufficient.

### CODE REVIEW

**Comment by jgosmann** [comment]
General question: Is there an easy way to look at the rendered versions of these PRs?

**Comment by tbekolay** [comment]
Not that I'm aware of, unfortunately. We could have TravisCI build it and put it up somewhere temporary, but I'm not sure where... For the time being, we have to build the pages locally. At least it doesn't take forever to build like the Nengo docs....

**Comment by hunse** [comment]
Is the plan to move the code documentation here too, or will that stay on pythonhosted.org (which has been down a bit lately)?

**Comment by Seanny123** [comment]
> Is the plan to move the code documentation here too, or will that stay on pythonhosted.org (which has been down a bit lately)?

We talked about doing this in #10 and from that discussion it does appear to still be the plan.

File: build-a-brain/index.rst [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+It exploits the `Neural Engineering Framework (NEF) <NEF>`_
+to develop the Semantic Pointer Architecture (SPA) for cognitive modelling.
+The book uses Nengo to explain and demonstrate
+many of the central concepts for these frameworks.
+This section of the website supports the book by providing
+links to models, demos, videos, and tutorials mentioned in the book.
+
+.. seealso::
+   * `The Nengo SPA project <https://nengo.github.io/nengo_spa>`_.
+   * `Spaun documentation <https://xchoo.github.io/spaun2.0>`_.
```
Comment (Seanny123) [inline build-a-brain/index.rst]: Might also want to include a link to the Nengo 2.0 version of the tutorials?

File: build-a-brain/index.rst [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+Let us consider how the model would run
+on one run of one cognitive task
+(`see a video of Spaun performing this task <spaunvideos>`_).
+This task is analogous to the
+`>Raven's Matrix task
+<http://en.wikipedia.org/wiki/Raven's_Progressive_Matrices>`_,
+which requires people to figure out a pattern in the input,
+and apply that pattern to new input to produce novel output.
+For instance given the following input
+"[1] [11] [111] [2] [22] [222] [3] [33] ?"
```
Comment (Seanny123) [inline build-a-brain/index.rst]: Might be possible to format this better using MathJax, but maybe that's annoying?

File: build-a-brain/index.rst [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+are implemented by a biologically plausible basal ganglia model.
+This is one example of the 8 different tasks that Spaun is able to perform.
+Videos for all tasks can be found `here <spaunvideos>`_.
+
+Book reviews
+============
+
+Here is a list of reviews we've found
+that talk about the book
+(feel free to send others you find):
```
Comment (Seanny123) [inline build-a-brain/index.rst]: Might want to specify via link who to send them to?

File: build-a-brain/supplementary.rst [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+Post synaptic time constants for various neurotransmitters
+with supporting citations can be found on `this page
+<http://compneuro.uwaterloo.ca/research/constants-constraints/neurotransmitter-time-constants-pscs.html>`_.
+
+Chapter 3
+=========
+
+The 6 DOF physical arm model can be `downloaded here
+<http://compneuro.uwaterloo.ca/research/motor-control/6-muscle-3-link-arm-model.html>`_.
```
Comment (Seanny123) [inline build-a-brain/supplementary.rst]: The "research" pages of the website are a bit ephemeral. Might want to move this link to somewhere safer?

File: build-a-brain/textbook.rst [hunk 5]
```diff
@@ -0,0 +1,157 @@
...
+
+The assignments above mirror those in the book.
+It is also useful to ask questions
+about the functioning of the models being built,
+and to ask students to change the models slightly
+to examine different aspects of neural/cognitive processing.
+The intent of this section of the website
+is to provide examples of such questions,
+as we develop them.
+Suggestions welcome.
```
Comment (Seanny123) [inline build-a-brain/textbook.rst]: Might want to add a link again to tell people where to submit suggestions? Would a link to the forum be of use?

File: build-a-brain/textbook.rst [hunk 6]
```diff
@@ -0,0 +1,157 @@
...
+
+Suggested projects
```
Comment (Seanny123) [inline build-a-brain/textbook.rst]: Should we link to our "modelling ideas" repository here? Some of these suggestions are out of date and a bit weird.

File: build-a-brain/index.rst [hunk 7]
```diff
@@ -0,0 +1,197 @@
...
+from Oxford University Press came out in May 2013.
+It exploits the `Neural Engineering Framework (NEF) <NEF>`_
+to develop the Semantic Pointer Architecture (SPA) for cognitive modelling.
+The book uses Nengo to explain and demonstrate
+many of the central concepts for these frameworks.
+This section of the website supports the book by providing
+links to models, demos, videos, and tutorials mentioned in the book.
+
+.. seealso::
+   * `The Nengo SPA project <https://nengo.github.io/nengo_spa>`_.
```
Comment (jgosmann) [inline build-a-brain/index.rst]: That page currently does not exist. Maybe we should link to the actual GitHub repo for now?

**Comment by jgosmann** [comment]
I was starting to review this PR, but came across a number of broken links and wasn't sure which of those will be fixed by the other PRs. So I'll wait until these PRs are merged before taking a look at this PR.

File: build-a-brain/index.rst (hunk 2 above)
Comment (tbekolay) [inline build-a-brain/index.rst]: If we're going to change this string of characters, I would probably want to change all of the characters that correspond to Spaun inputs... and I'm not sure if MathJax or `monospace` would look better? If it's ok with you, let's copy this over as it was on nengo.ca first and fix up the formatting later?

File: build-a-brain/index.rst (hunk 3 above)
Comment (tbekolay) [inline build-a-brain/index.rst]: Done, pointed to the forum

File: build-a-brain/supplementary.rst (hunk 4 above)
Comment (tbekolay) [inline build-a-brain/supplementary.rst]: Hmm... right now the only places that I can find this are on the research pages and the model archive -- it's not a Nengo model though, so I'm not sure if that'll get moved along with it... will talk to @studywolf about where to put it more permanently, so I just added a todo comment for now.

File: build-a-brain/textbook.rst (hunk 5 above)
Comment (tbekolay) [inline build-a-brain/textbook.rst]: Done!

File: build-a-brain/textbook.rst (hunk 6 above)
Comment (tbekolay) [inline build-a-brain/textbook.rst]: Sure why not!

**Pull request merged**